src/aggregate.d

```diff
@@ -126,7 +126,7 @@ public:
             sc2.inunion = 1;
         sc2.protection = Prot(PROTpublic);
         sc2.explicitProtection = 0;
-        sc2.structalign = STRUCTALIGN_DEFAULT;
+        sc2.aligndecl = null;
```

Comment:
Reasons for pulling things are never "why not". They are always "why". Constantly changing names needs stronger reasons because of the downsides of churn.